### PR TITLE
test(Wallet) Settings - account reordering not possible with 1 account

### PR DESCRIPTION
**Pull request by Valentina1133**
https://ethstatus.testrail.net/index.php?/cases/view/703416

Screenshot 2023-09-15 at 11 42 47

### CODE REVIEW

File: gui/screens/settings.py [hunk 1]
```diff
@@ -306,3 +316,7 @@ def drag_account(self, name: str, index: int):
         bounds = [account for account in self.accounts if account.name == name][0].object.bounds
         d_bounds = self.accounts[index].object.bounds
         driver.mouse.press_and_move(self._accounts_list.object, bounds.x, bounds.y, d_bounds.x, d_bounds.y)
+
+    @allure.step('Verify that back button is present')
+    def check_back_button_present(self) -> bool:
```
Comment (StateOf-Vlado) [inline gui/screens/settings.py:321]: is_back_button_present

File: gui/screens/settings.py [hunk 2]
```diff
@@ -268,6 +268,16 @@ def __init__(self):
         super(EditAccountOrderSettings, self).__init__()
         self._account_item = QObject('settingsContentBaseScrollView_draggableDelegate_StatusDraggableListItem')
         self._accounts_list = QObject('statusDesktop_mainWindow')
+        self._text_on_top = QObject('settingsContentBaseScrollView_StatusBaseText')
+        self._back_button = Button('main_toolBar_back_button')
+
+    @property
+    @allure.step('Get text labels')
+    def text_labels(self):
```
Comment (StateOf-Vlado) [inline gui/screens/settings.py:276]: Rename it to understand which labels you collect here
All labels from the settings view?
When I open it how can I use it?

Comment (Valentina1133) [reply gui/screens/settings.py]: I collect here all text labels from Edit order account settings. I renamed how I see it. Is that ok now?

File: gui/screens/settings.py (hunk 1 above)
Comment (Valentina1133) [reply gui/screens/settings.py]: Fixed

File: gui/screens/settings.py [hunk 3]
```diff
@@ -268,6 +268,16 @@ def __init__(self):
         super(EditAccountOrderSettings, self).__init__()
         self._account_item = QObject('settingsContentBaseScrollView_draggableDelegate_StatusDraggableListItem')
         self._accounts_list = QObject('statusDesktop_mainWindow')
+        self._text_item = QObject('settingsContentBaseScrollView_StatusBaseText')
+        self._back_button = Button('main_toolBar_back_button')
+
+    @property
+    @allure.step('Get all edit account order text labels')
+    def text_labels_from_edit_account_order_settings(self):
```
Comment (StateOf-Vlado) [inline gui/screens/settings.py:276]: Discussed in private chat and decided to rename it to get_account_recommendations, if it is a property - account_recommendations

Comment (Valentina1133) [reply gui/screens/settings.py:276]: Done

**Review by StateOf-Vlado — APPROVED** [review]

**Pull request merged**